modules/ROOT/pages/cassandra/cassandra-config-topics.adoc

```diff
@@ -0,0 +1,27 @@
+= Configuration Topics - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Cassandra DB connector configuration must match the Cassandra configuration with regard to Basic Authentication.
+By default, accessing the Cassandra database does not require credentials, so you need to configure Cassandra to enable authentication.
+After enabling authentication, run the `cqlsh` command from the Cassandra installation home directory as shown in this example. It uses default credentials with the username cassandra and password cassandra:
```

Comment:
Should the username and password be bolded?

Reply:
I don't know the rules around this. What does the Salesforce style guide say?

Reply:
Will bold unless I hear otherwise.